### ISSUE TITLE
Implement new DL Algorithms

### ISSUE THREAD

**Issue by HemanthSai7** [Math-behind-AI/ScratchAI#11]
**Name**
- [ ] Artificial Neural Network
- [ ] Adaline Neural Network
- [x] Multilayer Perceptron
- [x] Convolutional neural networks
- [ ] Cross Entropy Loss
- [ ] Stochastic gradient descent
- [x] Adam Optimizer
- [ ] RMS Prop
- [ ] Recurrent Neural Networks
- [ ] Vision Transformers
- [ ] Transformers
- [x]  Word embeddings
- [ ] LSTM

**Packages used**
List all the packages and versions used.

**Brief  explanation**
Provide a neat README file in the directory. There are two existing algorithms in the repository, please follow a similar folder structure and document your code with proper markdown syntax/comments.

**Comment by Shreyashmohadikar** [comment]
I would like to contribute by adding implementation of CNN from scratch
Please assign the issue to me 
Thank you

**Comment by HemanthSai7** [comment]
Hey @Shreyashmohadikar,
Thank you for your interest in ScratchAI. Assigning this issue to you. Feel free to reach out if you have any queries. Once you are done with the implementation make a PR.

**Comment by Vice777** [comment]
Hi, 
I would like to contribute by adding implementation of Multilayer Perceptron from scratch
Please assign the issue to me
Thank you

**Comment by mayureshagashe2105** [comment]
Hey @Vice777 ,
Thank you for your interest in ScratchAI. Assigning this issue to you. Feel free to reach out if you have any queries. Once you are done with the implementation make a PR.

**Comment by ag-yash** [comment]
I would like to contribute by adding ANN from scratch. Please assign this issue to me. I am a Hacktoberfest Contributor :)

**Comment by anju-chhetri** [comment]
I would like to contribute by implementing Stochastic gradient descent.
Please assign the issue to me.
Thank you

**Comment by mayureshagashe2105** [comment]
Hey @anju-chhetri ,
Thank you for your interest in ScratchAI. Assigning this issue to you. Feel free to reach out if you have any queries. Once you are done with the implementation make a PR.

**Comment by r-o-n-i-t** [comment]
I would like to contribute by adding the implementation of Adam Optimizer from scratch
Please assign the issue to me
Thank you
@mayureshagashe2105 
@HemanthSai7

**Comment by mayureshagashe2105** [comment]
@r-o-n-i-t , Sure go ahead. Assigning it to you!

**Comment by HemanthSai7** [comment]
@mayureshagashe2105 I'll work on Adaline neural network.

**Comment by mayureshagashe2105** [comment]
@HemanthSai7 go ahead!

**Comment by HemanthSai7** [comment]
Hola contributors,
Since tomorrow is the last day of HacktoberFest, if anyone still wants to create a PR and make it count as a valid PR in Hacktober you are most welcome.🚀Happy open source!
Thanks.

**Comment by cybergeekgyan** [comment]
I would like to work on Image Recognition using Deep Learning based Transfer learning and Vision Transformers(ViT).
Can you add me to the assignee @HemanthSai7

**Comment by mayureshagashe2105** [comment]
@cybergeekgyan , feel free to open a PR once you are done with the implementations. Thank you.
